sample/kotlin/src/main/kotlin/com/datadog/android/sample/SampleApplication.kt

```diff
@@ -117,13 +118,22 @@ class SampleApplication : Application() {
         Datadog.setVerbosity(Log.VERBOSE)
 
         val sessionReplayConfig = SessionReplayConfiguration.Builder().apply {
+            useSite(DatadogSite.valueOf(BuildConfig.DD_SITE_NAME))
             if (BuildConfig.DD_OVERRIDE_SESSION_REPLAY_URL.isNotBlank()) {
                 useCustomEndpoint(BuildConfig.DD_OVERRIDE_SESSION_REPLAY_URL)
             }
         }.build()
         val sessionReplayFeature = SessionReplayFeature(sessionReplayConfig)
         Datadog.registerFeature(sessionReplayFeature)
 
+        val logsFeature = LogsFeature.Builder().apply {
```

Comment:
why do we have 2 different way of building a feature ? I see that SR feature is built by passing a config to its constructor whereas for Logs you have a builder instead. Can't we just make them the same ?

Reply:
Yes, that is the plan - make SR follow the same pattern later.